src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,405 @@
+See [Common image-derived labels](#common-image-derived-labels) for a description
+of how integer values map to anatomical structures.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat|func|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>]_dseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+            sub-001_space-orig_dseg.json
+```
+
+A segmentation can be used to generate a binary mask that functions as a
+discrete "label" for a single structure.
+In this case, the mask suffix MUST be used, the `label` entity SHOULD be used
+to specify the masked structure
+(see [Common image-derived labels](#common-image-derived-labels)),
+and the `Atlas` metadata SHOULD be defined.
+For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-GM_mask.nii.gz
+```
+
+### Probabilistic Segmentations
+
+Probabilistic segmentations of brain tissue represent a single anatomical
+structure with values ranging from 0 to 1 in individual 3D volumes or across
+multiple frames.
+If a single structure is included, the `label` entity SHOULD be used to specify
+the structure.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_label-<label>]_probseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-BG_probseg.nii.gz
+            sub-001_space-orig_label-WM_probseg.nii.gz
+```
+
+See [Common image-derived labels](#common-image-derived-labels)
+for reserved key values for `label`.
+
+A 4D probabilistic segmentation, in which each frame corresponds to a different
+tissue class, must provide a label mapping in its JSON sidecar. For example:
+
+```Text
+pipeline/
+    sub-001/
+	    anat/
+		    sub-001_space-orig_probseg.nii.gz
+		    sub-001_space-orig_probseg.json
+```
+
+The JSON sidecar MUST include the label-map key that specifies a tissue label
+for each volume:
+
+```JSON
+{
+    "LabelMap": [
+        "BG",
+        "WM",
+        "GM"
+        ]
+}
+```
+
+Values of `label` SHOULD correspond to abbreviations defined in
+[Common image-derived labels](#common-image-derived-labels).
+
+### Discrete surface segmentations
+
+Discrete surface segmentations (sometimes called *parcellations*) of cortical
+structures MUST be stored as GIFTI label files, with the extension `.label.gii`.
+For combined volume/surface spaces, discrete segmentations MUST be stored as
+CIFTI-2 dense label fles, with the extension `.dlabel.nii`.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat/
+            <source_entities>[_hemi-{L|R}][_space-<space>][_res-<label>][_den-<label>]_dseg.{label.gii|dlabel.nii}
+```
+
+The `hemi` tag is REQUIRED for GIFTI files. For example:
```

Comment:
What is the rationale for requiring the `hemi` tag? I'm having a very quick read so sorry if I missed something obvious.

Reply:
I believe it's because GIFTI files are generally single-hemisphere while CIFTI-2 files generally contain both hemispheres in a single file.

I'm a little bit torn on this one. On the one hand, you *can* put both hemispheres in a single GIFTI file. On the other hand, I don't know that any tooling exists to handle those files.

In the end, I do think this probably should be RECOMMENDED.

Reply:
how about: ```The `hemi` tag is REQUIRED for GIFTI files storing information about a structure that is restricted to a hemibrain```.

Reply:
There are definitely GIfTI files containing bilateral structures. I would agree with RECOMMENDED.

Reply:
Sure.

```suggestion
The `hemi` tag is REQUIRED for GIFTI files storing information about
a structure that is restricted to a hemibrain.
For example:
```

Reply:
Sorry @gllmflndn, GitHub didn't show me your comment when I was making mine. Is this change better, or would you prefer still reducing to RECOMMENDED?

Reply:
And I only saw @satra's afterwards too. I am happy with your change.